data/campaigns/The_Hammer_of_Thursagan/scenarios/09_Forbidden_Forest.cfg

```diff
@@ -292,36 +274,73 @@
             id=Angarthing
             x,y=40,17
         [/filter]
+        [fire_event]
+            name=Angarthing message
+        [/fire_event]
+    [/event]
+
+    [event]
+        name=Aiglondur message
+        first_time_only=yes
+        [message]
+            speaker=Aiglondur
+            message= _ "Rally on me, everyone. We have better things to do than brawl with elves."
+        [/message]
+    [/event]
 
+    [event]
+        name=Angarthing message
+        first_time_only=yes
         [message]
             speaker=Angarthing
             message= _ "Kal Kartha should be only another day’s march east of here."
         [/message]
+    [/event]
 
+    [event]
+        name=moveto
+        first_time_only=no
+        [filter]
+            # The signpost and the hexes adjacent to it.
+            x=39,39,40,40,40
+            y=17,18,16,17,18
```

Comment:
you can use x,y= for the signpost, and radius=1

One thing I would like to note.
If two event's may fire at the same time, they are fired in the way they are listed in the file. But this may change in the future (has been discussed in 1.13 and been delayed), best to write events in a way that it doesn't matter in which order they are fired.
In this case we do have multiple moveto events, but it doesn't mater :+1: